docs-ref-conceptual/azure-cli-vm-tutorial-4.md

```diff
@@ -21,11 +21,11 @@ Now that a VM has been created, detailed information about it can be retrieved.
 az vm show --name $vmName --resource-group $resourceGroup
 ```
 
-You'll see a lot of information, which can be difficult to parse visually. The returned JSON contains information on authentication, network interface storage,
+You see numerous information, which can be difficult to parse visually. The returned JSON contains information on authentication, network interface storage,
```

Comment:
"You see numerous information, which can be difficult to parse visually." needs a complete rewrite.  What are we trying to convey here?  A:  "Virtual Machines have many properties which can be difficult to parse visually."